agent/php_memcached.c

```diff
@@ -0,0 +1,18 @@
+#include "php_memcached.h"
+#include "nr_datastore_instance.h"
+#include "php_agent.h"
+
+nr_datastore_instance_t* nr_php_memcached_create_datastore_instance(
+    const char* host,
+    zend_long port) {
+  nr_datastore_instance_t* instance = NULL;
+  if (port == 0) { // local socket
+    instance = nr_datastore_instance_create("localhost", host, NULL);
+  } else {
+    char* port_str = nr_formatf("%ld", (long)port);
```

Comment:
`nr_formatf` seems a bit heavy handed for `itoa` conversion. FYI: there exists `nr_itoa`. Also can `zend_long` be safely cast to `long`?

Reply:
Looking at other places (e.g. [here](https://github.com/newrelic/newrelic-php-agent/blob/R11.1/agent/lib_predis.c#L187)), this seems ok.